modules/odc-access-web-terminal.adoc

```diff
@@ -23,6 +23,7 @@ The web terminal remains open until you close it or until you close the browser
 +
 [NOTE]
 ====
+* A single DevWorkspace CR defines the web terminal of a single user. The web terminal status and container image components are referenced from this CR.
```

Comment:
```suggestion
* One `DevWorkspace` CR defines the web terminal of one user. The web terminal status and container image components are referenced from this CR.
```
If possible, it would be best to eliminate the passive voice in the second sentence ("are referenced").

Reply:
Updated to:
```
One `DevWorkspace` CR defines the web terminal of one user.
This CR contains details about the user's web terminal status and container image components.
```